HEN_HOUSE/scripts/bin/egs-parallel

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
+###############################################################################
+#
+#  EGSnrc script to submit parallel jobs
+#  Copyright (C) 2020 National Research Council Canada
+#
+#  This file is part of EGSnrc.
+#
+#  EGSnrc is free software: you can redistribute it and/or modify it under
+#  the terms of the GNU Affero General Public License as published by the
+#  Free Software Foundation, either version 3 of the License, or (at your
+#  option) any later version.
+#
+#  EGSnrc is distributed in the hope that it will be useful, but WITHOUT ANY
+#  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+#  FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for
+#  more details.
+#
+#  You should have received a copy of the GNU Affero General Public License
+#  along with EGSnrc. If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+#
+#  Author:          Frederic Tessier, 2020
+#
+#  Contributors:
+#
+###############################################################################
+
+
+### help function
+function help {
+    log "HELP"
+    cat <<EOF
+
+    usage:
+
+        $(basename $0) [options] -c | --command 'command'
+
+    options:
+
+        -h | --help         show this help
+        -b | --batch        batch system to use ("cpu" by default)
+        -d | --delay        delay in seconds between individual jobs
+        -q | --queue        scheduler queue ("long" by default)
+        -n | --nthread      number of threads ("8" by default)
+        -o | --option       option(s) to pass to job scheduler, in quotes
+        -f | --force        force run, even if lock or egsjob file present
+        -v | --verbose      echo detailed egs-parallel log messages to terminal
+        -c | --command      command to run, given in quotes
+
+    example:
+
+        $(basename $0) --batch cpu -q short -n12 -v -c 'egs_chamber -i slab -p 521icru'
```

Comment:
inconsistent use of Pipe, example does not have pipe in it.

Reply:
These are not pipes but option "ORs". I think this is pretty standard and I find it clear (except in the example usage line, for sure!).